src/webgpu/api/validation/queue/copyToTexture/CopyExternalImageToTexture.spec.ts

```diff
@@ -124,6 +124,7 @@ function canCopyFromContextType(contextName: string) {
     case 'experimental-webgl':
     case 'webgl':
     case 'webgl2':
+    case 'webgpu':
```

Comment:
I just noticed that `source_offscreenCanvas,contexts` does not test `'webgpu'`. Let's deduplicate the list of context types and use it in both tests so they both stay up to date.